contracts/optimistic-ethereum/iOVM/bridge/ERC20/iOVM_L2ERC20Gateway.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+pragma experimental ABIEncoderV2;
+
+import { iOVM_ERC20 } from "../../precompiles/iOVM_ERC20.sol";
+
```

Comment:
Nit: extra spacing here when there is no extra spacing in the other contract directly below the imports. Consistency makes things easier to read/audit